SPECS-EXTENDED/v4l-utils/v4l-utils.spec

```diff
@@ -48,16 +53,6 @@ Requires:       libv4l%{?_isa} = %{version}-%{release}
 Utilities for v4l2 / DVB driver authors: decode_tm6000, v4l2-compliance and
 v4l2-dbg.
 
-
-%package -n     qv4l2
```

Comment:
Is removing this sub-package related to the build error fix?

Reply:
From Jyoti: I checked the dependency. There is no dependency on this subpackage qv4l2. [{_datadir}/applications/qv4l2.desktop]  it is a GUI subpackage ,which is directly related to frontend.